### ISSUE TITLE
User Defined Functions are not added to the database

### ISSUE THREAD

**Issue by molivasdat** [liquibase/liquibase-snowflake#117]
Snowflake recently added the ability to create user defined functions in a snowflake database in June of this year.
https://www.snowflake.com/blog/new-snowflake-features-released-in-june-and-july-2021/

**Extension Version**: 4.4.2 (4.4.3)
**Liquibase Version**: 4.4.2 (4.4.3)
**Integration**: CLI
**Snowflake Driver**: 3.13.6
**Driver**: net.snowflake.client.jdbc.SnowflakeDriver

Steps to recreate:
Use the following files
liquibase.properties:

```
url: jdbc:snowflake://<yourconnection>.snowflakecomputing.com/?db=mydb&schema=MYSCHEMA
username: youruser
driver: net.snowflake.client.jdbc.SnowflakeDriver 
classpath: ./snowflake-jdbc-3.13.6.jar:./liquibase-snowflake-4.4.2.jar
```

changelog file (changelog.sql)

```
--liquibase formatted sql
--changeset author1:002
create or replace function pi_udf()
  RETURNS FLOAT
  AS '3.141592654::FLOAT'
  ;
```

run
liquibase --changeLogFile=changelog.sql --password=yourpassword update

Or you can create a private/public key pair and pass that on the jdbc url in the properties file by adding &private_key_file=/yourpathtokey/rsa_key.p8 to the end instead of password
This requires setup of public private key pair to do so.
https://docs.snowflake.com/en/user-guide/key-pair-auth.html

┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-2040) by [Unito](https://www.unito.io)

**Comment by FBurguer** [comment]
Looks like it got fixed (at least for 4.12)

**Comment by molivasdat** [comment]
Agree, resolved in 4.12 possibly earlier.
